chasm/visibility.go

```diff
@@ -45,8 +35,6 @@ type Visibility struct {
 
 	SA   Map[string, *commonpb.Payload]
 	Memo Map[string, *commonpb.Payload]
```

Comment:
Seems like overkill to use `chasm.Map` for search attributes and arguably for memo too. The total memo size shouldn't exceed the size of a payload AFAIR.

Reply:
Synced offline, looks like the overall size will always be small, so partial write doesn't worth it. Will switch to just one payload.